### ISSUE TITLE
2016.1120.317-gamma002 release

### ISSUE THREAD

**Issue by blairconrad** [blairconrad/FakeItEasy#79]
Can be labelled **ready** when all other issues destined for this release are closed.

- [ ] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [ ] if necessary, change `VERSION_SUFFIX` in [AppVeyor](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings) to "-%suffix%" and initiate a build
- [ ] check build
- edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [ ] ensure completeness of release notes, including non-owner contributors, if any
    - [ ] attach main nupkg and analyzer nupkg
    - [ ] publish the release
- [ ] push nupkgs to NuGet
- [ ] tweet, mentioning contributors
- [ ] post link to tweet as comment here for easy retweeting ;-)
- [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [ ] if another pre-release is planned, run `rake pre_release[new_version]` to
    - create a new draft GitHub Release
    - create a new issue (like this one) for the next release, adding it to the milestone for the next production release
- [ ] close this issue
